R/gdal_utils.R

```diff
@@ -112,6 +112,7 @@ gdal_utils = function(util = "info", source, destination, options = character(0)
 						"-te", "-tr", "-tap", "-ts", "-ot")) # https://gdal.org/programs/gdal_rasterize.html
 				CPL_gdalrasterize(source, destination, options, oo, doo, config_options, overwrite, quiet)
 			}, # nocov end
+			gdalfootprint = CPL_gdalfootprint(source, destination, options, oo, config_options, quiet),
```

Comment:
I'd suggest to call the util `footprint`, rather than `gdalfootprint`, in line with the others. Also above in the doc section.

Reply:
Done!